string-manipulation-3/src/title-case.js

```diff
@@ -0,0 +1,49 @@
+/* exported titleCase */
+/*
+function with one parameter
+lowercase the full string to make it easier to capitalize
+add all the special case words into an array to hold onto
+get each individual word by splitting it
+upper case the first letter of each word and add the rest of the letters of that word
+change Javascript to JavaScript
+loop through the words
+-if its a special case you uppercase it
+-if it has a colon or dash you upper case the word prior
+if its api return API
+put the words together by the spaces they had
+*/
+
+function titleCase(title) {
+  var lowerCased = title.toLowerCase();
+  var specialCases = ['and', 'or', 'nor', 'but', 'a', 'as', 'at', 'by', 'for', 'in', 'of', 'on', 'per', 'to', 'the'];
+  var words = lowerCased.split(' ');
+  words[0] = words[0].charAt(0).toUpperCase() + words[0].substring(1);
+  if (words[0].charAt(0) === 'J') {
```

Comment:
Any title with a first word starting with `J` will now have that word replaced with "JavaScript"